pkgs/by-name/rt/rtkit/package.nix

```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
 
     (fetchpatch {
       name = "rtkit-daemon-dont-log-debug-messages-by-default.patch";
-      url = "https://github.com/heftig/rtkit/pull/33/commits/ad649ee491ed1a41537774ad11564a208e598a09.patch";
+      url = "https://github.com/heftig/rtkit/pull/33/commits/4880b9c67628a781bdb183dcdc69f12cb829817d.patch";
```

Comment:
This should be added locally, see https://github.com/NixOS/nixpkgs/tree/master/pkgs#patches